src/Esfa.Vacancy.Domain/Validation/ErrorMessages.cs

```diff
@@ -33,5 +33,11 @@ public static class GetTraineeship
         {
             public const string VacancyReferenceNumberNotNumeric = "The vacancy reference number must be numeric.";
         }
+
+        public static class CreateApprenticeship
+        {
+            public const string ApplicationClosingDateBeforeTomorrow = "'Application Closing Date' must be after today's date.";
+            public const string ExpectedStartDateBeforeClosingDate = "'Expected Start Date' must be after the specified closing date.";
```

Comment:
"'Expected Start Date' must be after the specified **application** closing date.";